chapter-7.qmd

```diff
@@ -0,0 +1,11 @@
+---
+title: "Workflow: scripts and projects"
+---
+
+
+
+## Solutions
+
+1. What tip from twitter did i like?
+
+If you go to tools-global options - appearance you can change the background.
```

Comment:
```suggestion
If you go to Tools > Global options > Appearance, you can change the background.
```

Reply:
I would add under which dropdown menu you can do that. And note that it's not just the background. You're changing the theme. So more accurate to say "you can change the theme".